scripts/linters/pre_commit_linter.py

```diff
@@ -508,11 +530,22 @@ def main(args=None):
     # third party libraries have their own ways to lint at their fastest
     # (ie. might parallelize on their own)
 
+    err_messages = []
```

Comment:
make it error_messages (for consistency)

Reply:
Done.